languages/csharp/exercises/concept/arrays/.docs/introduction.md

```diff
@@ -0,0 +1,45 @@
+In C#, data structures that can hold zero or more elements are known as _collections_. An **array** is a collection that has a fixed size and whose elements must all be of the same type. Elements can be assigned to an array or retrieved from it using an index, with the first element's index being zero:
```

Comment:
Apologies, I thought I commented this before, but I don't see it on history now.
People transitioning from other languages may be scanning for specific terms (zero-based), and newbies will still need the explanation.

```suggestion
In C#, data structures that can hold zero or more elements are known as _collections_. An **array** is a collection that has a fixed size and whose elements must all be of the same type. Elements can be assigned to an array or retrieved from it using an index. C# arrays are zero-based, meaning that the first element's index is always zero:
```